examples/pylint/C0102_blacklisted_name.py

```diff
@@ -0,0 +1,14 @@
+"""pylint: blacklisted name
+
+"""
+
+def pos(obj):
```

Comment:
I mention clearly about that in the website documentation. Hence, if I mention the same in the examples as well I feel it would be redundant. Any thoughts?